### PR TITLE
AV: Fix HCM for guidance background colour

### LINKED ISSUE #2572: AV: A11y HCM issue with Guidance in psammead-media-player
**Is your feature request related to a problem? Please describe.**
In the dev a11y swarm done for #2517, a few issues were found as documented in this [comment](https://github.com/bbc/psammead/pull/2517#issuecomment-549826195):

One of these issues is with Windows HCM. Currently, with the implementation of guidance, the current opaque overlay is causing the placeholder image to not be visible which it should be in High Contrast Mode.

**Describe the solution you'd like**
Our implementation for the overlay is using `background-color: rgba(34,34,34,0.75);`. From a brief bit of investigation, it looks like `background-color` in Windows HCM is being replaced by a block colour hence the placeholder image not appearing. 

We need to find a way to implement the opaque layer without blocking the placeholder image behind it in HCM.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by HarveyPeachey**
Resolves #2572

**Overall change:** Fixed a11y issue with image placeholder not showing when in High Contrast Mode in Windows for IE11 and Firefoxes colour preferences.

**Code changes:**

- Added media query to make background transparent when high contrast mode is enabled.
- Added transparent border so it's viewable using Firefox's colour preferences.
---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

**Comment by HarveyPeachey** [comment]
Spoke to @greenc05 and she's happy with the HCM changes for guidance. She also discovered an issue with the play button icon on the `media-player` not adhering to custom colours set in windows HCM, I'll create a separate issue for this.

**Review by 12 — APPROVED** [review]
LGTM 👍 

Do we have an issue to pull this into Simorgh, or are you just going to create the PR for it once this has been merged?

**Comment by HarveyPeachey** [comment]
> LGTM 👍
> 
> Do we have an issue to pull this into Simorgh, or are you just going to create the PR for it once this has been merged?

I'll make a PR when this is merged, I just need to make sure [this](https://github.com/bbc/simorgh/pull/4583) merges before I bump it in Simorgh.

**Review by ryanmccombe — APPROVED** [review]

File: packages/components/psammead-media-player/src/Guidance/index.jsx
Comment (simonsinclair) [inline packages/components/psammead-media-player/src/Guidance/index.jsx:35]: It looks like this `@media` block is missing a closing brace.

File: packages/components/psammead-media-player/src/Guidance/index.jsx
Comment (simonsinclair) [inline packages/components/psammead-media-player/src/Guidance/index.jsx]: Does this 1px transparent border need to exist when HCM is _not_ active? It creates additional padding which skews the GEL spacing.

File: packages/components/psammead-media-player/src/Guidance/index.jsx
Comment (simonsinclair) [inline packages/components/psammead-media-player/src/Guidance/index.jsx]: Is flex box necessary here?

File: packages/components/psammead-media-player/src/Guidance/index.jsx
Comment (simonsinclair) [inline packages/components/psammead-media-player/src/Guidance/index.jsx]: Please see comment on 1px border above.

File: packages/components/psammead-media-player/src/Guidance/index.jsx
Comment (simonsinclair) [inline packages/components/psammead-media-player/src/Guidance/index.jsx]: Have we considered putting these styles into GuidanceMessage to avoid further nesting and another wrapping div?

**Review by simonsinclair — CHANGES_REQUESTED** [review]

File: packages/components/psammead-media-player/src/Guidance/index.jsx
Comment (HarveyPeachey) [reply packages/components/psammead-media-player/src/Guidance/index.jsx:35]: I pushed a change that's already fixed this👍

File: packages/components/psammead-media-player/src/Guidance/index.jsx
Comment (HarveyPeachey) [reply packages/components/psammead-media-player/src/Guidance/index.jsx]: This transparency fix isn't for HCM, this a fix for firefox's colour preferences which cannot be targeted with a media query. Without the border a user with colour preferences won't be able to distinguish where the video is:
Screenshot 2019-11-11 at 11 02 37
Screenshot 2019-11-11 at 11 01 50

File: packages/components/psammead-media-player/src/Guidance/index.jsx
Comment (HarveyPeachey) [reply packages/components/psammead-media-player/src/Guidance/index.jsx]: Yep, I felt the CSS would be too finicky by just applying it to the message. I felt the message should be just the text as it is currently contained in `<strong>` and then a container `content` which would make it easier to set HCM fixes.

File: packages/components/psammead-media-player/src/Guidance/index.jsx
Comment (HarveyPeachey) [reply packages/components/psammead-media-player/src/Guidance/index.jsx]: Ohh didn't notice I left that in there, will remove.

File: packages/components/psammead-media-player/src/Guidance/index.jsx
Comment (simonsinclair) [reply packages/components/psammead-media-player/src/Guidance/index.jsx]: Cool, ok! Can we use `0.0625rem` instead of `1px` here, please?

**Review by simonsinclair — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
LGTM..

**Comment by paruchurisilpa** [comment]
Moved to Ready to Code Review as it says Code owner review required..Can be moved straight to merge once the review happens as this has been tested..

**Review by j-pendlebury — APPROVED** [review]
LGTM

**Pull request merged**
